src/Compilers/CSharp/Portable/Lowering/AsyncRewriter/AsyncExceptionHandlerRewriter.cs

```diff
@@ -890,7 +884,7 @@ private sealed class AwaitFinallyFrame
             public readonly HashSet<LabelSymbol> LabelsOpt;
 
             // the try or using-await statement the frame is associated with
-            private readonly StatementSyntax _statementSyntaxOpt;
```

Comment:
The node associated with `BoundTryStatement` does not need to be a statement. It could, e.g. be a local variable declarator whose declaration specifies `using` (e.g. `using T x = new(), y = new()` produces two `BoundTryStatement`s, one for each declarator.